gym/envs/mujoco/mujoco_env.py

```diff
@@ -101,12 +101,22 @@ def do_simulation(self, ctrl, n_frames):
         for _ in range(n_frames):
             self.sim.step()
 
-    def render(self, mode='human', width=DEFAULT_SIZE, height=DEFAULT_SIZE):
+    def render(self,
+               mode='human',
+               width=DEFAULT_SIZE,
+               height=DEFAULT_SIZE,
+               camera_id=None,
```

Comment:
I think it's important to be able to specify the camera name for the pixel observation, which is why added `camera_id` and `camera_name` as arguments here.